evidence/brenzi/0003-2024-09-evidence.md

```diff
@@ -0,0 +1,61 @@
+# Evidence-0003: Retention at Rank 2
+
+|                 |                   |
+| --------------- |-------------------|
+| **Report Date** | 2024/09/20      |
+| **Submitted by**| Alain Brenzikofer |
+
+
+## Member details
+
+- Matrix username: `@brenzi:matrix.org`
+- Polkadot address: `12brenJreRTd6XsBMo3Ptu5VMUUycC53ipDxUy9Mvb4L1s1q`
+- Current rank: 2 ([promotion](https://collectives.subsquare.io/fellowship/referenda/139) executed on July 12th)
+- Date of initial induction: 2023/09/22 (induction for salary: 2024/02/22)
+- Date of last report: 2024/05/13
+- Area(s) of Expertise/Interest: Encointer systemchain (code owner), Polkadot governance
+
+
+## Reporting period
+
+- Start date: 2024/05/14
+- End date: 2024/09/20
+
+
+## Evidence
+
+Since my [last evaluation submission](./0002-2024-05-evidence.md), I have focused on advancing Encointer's democracy 
+experiments while also contributing to the maintenance of the Encointer runtime. A highlight of my work is [PR#456](https://github.com/polkadot-fellows/runtimes/pull/456) 
+which introduces one-person-one-vote (1p1v) experiments on the Encointer Network. This is a pioneering initiative 
+within the Polkadot ecosystem and holds potential for influencing alternative governance models, such as sub-treasury 
+management or petition systems on Kusama and Polkadot. I’m eager to see how this will unfold.
+
+While this achievement was a team effort, my role centered on providing conceptual leadership and specification. 
+I led the implementation of key features, enabling communities to allocate treasury funds using 1p1v democracy and 
+to support petitions. Additionally, I developed testing environments using agent-based e2e simulations on a local testnet 
+to validate the new democratic processes.
+
+Beyond my contributions to Encointer, I remained an active member of the community. I spoke at Decoded 2024 and 
+participated in the in-person fellowship meeting in Brussels.
+
+## Voting record
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities                                | Comments |
+|---|---|---|-----------------------------------------------------------|---|
+|I  |90%   |N/A   |  |  |
+|II |80%   |N/A   | No eligible referenda since my promotion to rank II       |  |
```

Comment:
Please keep in mind that Rank 2 members are eligible to vote on referenda posted on the `Members`  and `Proficient members` tracks.
This means that, as a Rank 2, you were eligible to vote on Ref [117](https://collectives.subsquare.io/fellowship/referenda/117) and [150](https://collectives.subsquare.io/fellowship/referenda/150).

Reply:
Seems I misunderstood your earlier comment in DM. Had it in there, then removed it, now put it back in...